patcher/Config.py

```
@@ -0,0 +1,70 @@
+
+        self.build_folder_path = Path(self.build_folder_path)\
+            if self.build_folder_path \
+            else self.target_folder_path / "build"
+
+        self.clang_path = Path(self.clang_path)
+        self.gcc_path = Path(self.gcc_path)
+        self.linker_path = Path(self.linker_path)
+
+        if not self.build_folder_path.is_absolute():
+            raise ValueError(
+                "build_folder_path must be an absolute path to folder")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)